src/test/java/com/at/globalclasses/MongoDBConnection.java

```diff
@@ -156,6 +162,72 @@ public boolean compareJsonString(String collection, String object) {
         return bool;
 
     }
+
```

Comment:
Let's move this method to MongoDBUtils class

Reply:
Ready.
I don't add this method but I already moved it.

Reply:
I didn't add this method to the code. I am trying to move it, but I have problems because there are other method with the same name in MONGO DB Utils, but it is only to implement the connection with the DB.